.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Please briefly describe your problem and what output you expect.
+
+Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](http://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.
+
+---
+
+Brief description of the problem
+
+```r
+# insert reprex here
+```
```

Comment:
```suggestion
```

## Session Information
<!-- Please provide session information in the code blocks below using the `sessionInfo()` command in your R console. -->

```console
# Insert Session info here
```

```